framework/src/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -302,7 +302,7 @@
     private final static int DEFAULT_CACHE_SIZE = 300;
 
     /** Mapping from a Tree to its annotated type; implicits have been applied. */
```

Comment:
The name suggests that the key must be a ClassTree or a MethodTree.  Is that true?  If so, I would so note in the Javadoc comment.